src/recovery/cubic.rs

```diff
@@ -218,10 +223,10 @@ fn on_packet_acked(
             cubic_cwnd = cmp::max(cubic_cwnd, w_est as usize);
         } else if cubic_cwnd < w_cubic as usize {
             // Concave region or convex region use same increment.
-            let cwnd_inc = (w_cubic - cubic_cwnd as f64) / cubic_cwnd as f64 *
+            let cubic_inc = (w_cubic - cubic_cwnd as f64) / cubic_cwnd as f64 *
```

Comment:
Why rename this?

Reply:
Because I added `cwnd_inc` to `cubic_state` to store the cwnd increments, so try to avoid to use same name. Also `cubic_inc` is better because here it mean cwnd increments by CUBIC algorithm.